src/logging.rs

```diff
@@ -18,6 +18,10 @@ pub fn setup_logging(cfg: &Configuration) {
         },
     };
 
+    if log_level == log::LevelFilter::Off {
```

Comment:
hi @da2ce7 @WarmBeer, I've changed the `setup_logging` function to avoid panicking if it's called more than once. This way, we can enable logging even for testing (using more than one UdpServe). I still use the "off" level in testing because I prefer to keep the test output clean. You can enable it whenever you needed it.